### ISSUE TITLE
Accessibility Error - Form elements need a label

### ISSUE THREAD

**Issue by iantrudell** [ng-select/ng-select#407]
### Expected behaviour
https://dequeuniversity.com/rules/axe/2.2/label?application=lighthouse

Form elements like input need a label or they will fail the Chrome Lighthouse Accessibility test

### Actual behaviour
![form-elements](https://user-images.githubusercontent.com/33202657/38197194-1ce5bdc0-3655-11e8-9a6f-5edd2f990b92.PNG)

**Comment by varnastadeus** [comment]
I think this should be addressed to #216 which should fix all accessibility issues.

**Comment by bhuvanabellala1** [comment]
I have a simple ng-select with ng-label-tmp and ng-option-tmp. I want to place an aria-label on the inputbox. Tried following the example but to no avail. How do I place an aria attribute on the combobox?

**Comment by esoyke** [comment]
This seems to be screen-reader dependent. Using both JAWS and NVDA readers, it was reading my custom aria-label on the ng-select properly. But using Apple's VoiceOver, it merely reads a generic 'Menu popup combo box'.
